### ISSUE TITLE
[1] a11y: improve clarity of messaging of verifiable fields for screen readers

### ISSUE THREAD

**Issue by karrui** [opengovsg/FormSG#4188]
Users could be confused by the notion of needing to verify their email/mobile numbers due to lack of context when they navigate to the verify button when using a screen reader.
Also applies to verification box suddenly closing (and the verify button becoming disabled) after verification success.

Suggest as a first cut add additional description to verifiable fields that will be announced after the question title and description:

**Phone number**
This is a verified field so after you enter your phone number, please click on the Verify button. You can then enter a six digit One-time password below, which you will receive in an SMS.

**Email**
This is a verified field so after you enter your email, please click on the Verify button. You can then enter a six digit One-time password below, which you will receive in your email.

**Comment by karrui** [comment]
some copy changes i think reads better:

thoughts, @amitogp, @syan-syan, @pearlyong, @mantariksh?

**Email**:
This is an input field which requires verification. After you enter the email address to be verified, please click on the Verify button. A one-time password will be sent to the entered email address, which you can then enter in the verification input field ~~that is shown during the verification step~~.

**Mobile**:
This is an input field which requires verification. After you enter the mobile phone number to be verified, please click on the Verify button. A one-time password will be sent via SMS to the entered number, which you can then enter in the verification input field ~~that is shown during the verification step~~.

**Comment by amitogp** [comment]
@karrui can we delete the last part?
`that is shown during the verification step`
verification input field should be sufficient.

**Comment by r00dgirl** [comment]
Thanks @karrui ! Suggesting amended copy below for brevity, but I think the original is okay too.

This is an input field which requires verification. After **entering your email address**, please click on the Verify button. **You will receive a one-time password, which you can** enter in the verification input field.
